web/tests/narrow_banner.test.cjs

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+const assert = require("node:assert/strict");
+
+const {$t} = require("./lib/i18n.cjs");
+const {mock_esm, zrequire} = require("./lib/namespace.cjs");
+const {run_test} = require("./lib/test.cjs");
+
+const stream_data = mock_esm("../src/stream_data");
+const {Filter} = zrequire("filter");
+const {pick_empty_narrow_banner} = zrequire("narrow_banner");
+
+run_test("pick_empty_narrow_banner for private stream", ({override}) => {
```

Comment:
This should be with the tests for other empty narrow banners, which is in "web/tests/message_view.test.cjs`.